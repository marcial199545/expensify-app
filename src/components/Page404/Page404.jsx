import React from "react";
import { Link } from "react-router-dom";
const Page404 = () => (
    <div>
        Page not found - <Link to="/">Go Home</Link>
    </div>
);
export default Page404;
