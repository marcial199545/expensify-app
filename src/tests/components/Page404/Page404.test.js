import React from "react";
import { shallow } from "enzyme";
import Page404 from "../../../components/Page404/Page404.jsx";

test("should render the Page404 component", () => {
    const wrapper = shallow(<Page404 />);
    expect(wrapper).toMatchSnapshot();
});
