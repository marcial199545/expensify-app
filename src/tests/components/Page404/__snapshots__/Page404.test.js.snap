// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Page404 component 1`] = `
<div>
  Page not found - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
